cloudinit/sources/DataSourceVultr.py

```diff
@@ -136,7 +137,16 @@ def get_datasource_list(depends):
         print("Machine is not a Vultr instance")
         sys.exit(1)
 
+    # It should probably be safe to try to detect distro via stages.Init(),
```

Comment:
@eb3095 I'm not quite sure what `__main__` gets used for, or if it is even still used at all. Would you mind looking at this section if you get a chance?

Basically piping distros throughout the dhcp and ephemeral setup code means that `vultr.get_metadata()` will now require a distro object. This approach uses the same strategy that cloud-init normally uses. Will that work here? Based on the `is_vultr()` check above, I assume that this should be running on an image with cloud-init. Therefore, I assume that this approach would work, but I want to double check my assumption.

Reply:
Yeah this should work fine. __main__ i believe was an early discussion about how to debug. I dont think we have ever really used it, but that was the intended purpose.